PlayCover/Views/Settings/UninstallSettings.swift

```diff
@@ -14,6 +14,7 @@ class UninstallPreferences: NSObject, ObservableObject {
     @objc @AppStorage("RemoveAppKeymapUninstall") var removeAppKeymap = false
     @objc @AppStorage("RemoveAppSettingUninstall") var removeAppSettings = false
     @objc @AppStorage("RemoveAppEntitlementsUninstall") var removeAppEntitlements = false
+    @objc @AppStorage("RemovePlayChain") var removePlayChain = false
```

Comment:
The key for the App Storage should have the `Uninstall` prefix to maintain the uninstall setting naming convention.